sympy/core/evalf.py

```diff
@@ -186,13 +210,13 @@ def scaled_zero(mag, sign=1):
         raise ValueError('scaled zero expects int or scaled_zero tuple.')
 
 
-def iszero(mpf, scaled=False):
+def iszero(mpf: tUnion[MPF_VALUE_TUPLE, SCALED_ZERO_TUPLE, None], scaled=False) -> Optional[bool]:
     if not scaled:
         return not mpf or not mpf[1] and not mpf[-1]
     return mpf and isinstance(mpf[0], list) and mpf[1] == mpf[-1] == 1
```

Comment:
Is the return type here not just `bool`?

Reply:
`iszero(None, True)` returns `None`. This may happen in https://github.com/sympy/sympy/blob/9019765edb0d0b4bbd29674a58b93ef4e75b92f8/sympy/core/evalf.py#L554